manifest.json

```diff
@@ -113,13 +113,17 @@
         "type": "string",
         "title": "entryFeature"
       },
-      "urlOne": {
+      "baseCoreURL": {
         "type": "string",
-        "title": "Url 1"
+        "title": "baseCoreURL"
       },
-      "urlTwo": {
+      "communityEndpointURL": {
         "type": "string",
```

Comment:
Ops, that's a breaking change. 

Do you want to keep this change and deploy a major version of the app?

Or do you want to use the previous name and keep the same major version?

Reply:
A new major version of the app will require stores that use this app to install the app again to use the new version.

Keeping the major version, the stores that use this app will get it automatically.

Reply:
Oh, you're right, best to keep the old names and keep the same major then.